src/cosmology/api/_distances.py

```diff
@@ -5,26 +5,31 @@
 from typing import Protocol, runtime_checkable
 
 from cosmology.api._array_api import ArrayT_co
-from cosmology.api._core import Cosmology, InputT_contra
+from cosmology.api._core import InputT_contra
 
 __all__: list[str] = []
 
 
-# This is currently private
-class _HasTcmb(Cosmology[ArrayT_co, InputT_contra], Protocol):
-    r"""The cosmology contains a CMB temperature, described by :math:`T_{CMB}`."""
+@runtime_checkable
+class HasTcmb0(Protocol[ArrayT_co]):
```

Comment:
An interesting edge case. I was camel-casing the underlying words since I though `HasOmegaDE` is more intuitive than `HasOmegaDe`, analogous to How We Demonstrate an Acronym (HWDA). So yes, I think it should be `HasTCMB0`.